### PR TITLE
refractored: responsive for mobile screen

### PR DESCRIPTION
**Pull request by JantuDeb**
#18 
# Product Filter
### Filter button for shwoing filters in mobile screen.
imageimage

### CODE REVIEW

**Review by fahd23 — APPROVED** [review]
Code looks clean

File: src/pages/Products.jsx [hunk 1]
```diff
@@ -1,21 +1,46 @@
 import React from "react";
+import { useState, useEffect } from "react";
 import Product from "../components/products/Product";
 import ProductFilter from "../components/products/ProductFilter";
 import TopBar from "../components/products/TopBar";
```
Comment (t007rushi) [inline src/pages/Products.jsx:5]: its increasing lines code I think you forgot to single line import from ```index.js```

File: src/components/products/product.css [hunk 2]
```diff
@@ -41,6 +42,63 @@
...
+@media only screen and (min-width: 600px) {
+  .category-card {
+    width: 12rem;
+  }
+  .product-filter-container {
+    margin-bottom: 0rem;
+    flex-direction: row;
+  }
+  .show-filter-bar{
+    display: none;
```
Comment (t007rushi) [inline src/components/products/product.css:94]: liked the small well written queries

**Review by t007rushi — APPROVED** [review]
Very responsive on S-size mobile devices great work

File: src/pages/Products.jsx (hunk 1 above)
Comment (JantuDeb) [reply src/pages/Products.jsx:5]: Thanks for the suggestion

**Pull request merged**